.github/workflows/build.yml

```diff
@@ -145,3 +145,49 @@ jobs:
           check_name: Route Service Test Result
           comment_mode: "create new"
           files: test/api-test/build/test-results/**/*.xml
+  cli-sandbox:
+    runs-on: ubuntu-latest
+    needs: [ build, changes ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Create Sandbox
+        env:
+          SIGNADOT_ORG: signadot
+          SIGNADOT_API_KEY: ${{ secrets.SIGNADOT_API_KEY }}
+          IMAGE_TAG: ${{ github.sha }}
+        run: |
+          # run the cli
+          # setting the gitsha to a short version to respect the name
+          # limit (30 chars)
+          docker run -i \
+            -e SIGNADOT_ORG=${SIGNADOT_ORG} \
+            -e SIGNADOT_API_KEY=${SIGNADOT_API_KEY} \
+            signadot/signadot-cli \
+            /signadot sandbox apply \
+            --set image=signadot/hotrod:${IMAGE_TAG} \
+            --set git-short-commit=${IMAGE_TAG:0:6} \
+            -f - \
+            -o yaml \
+            < ${GITHUB_WORKSPACE}/.signadot/frontend-template.yaml \
+            > ${GITHUB_WORKSPACE}/.signadot/frontend.yaml
```

Comment:
can we name this something that indicates that it's the output? It wasn't super clear at first pass.

Reply:
Trying to think of some alternatives: `frontend-sandbox.yaml`, `frontend-applied.yaml`, `frontend-output.yaml`.  

(Without an extension, this also plays off the description at https://deploy-preview-57--signadot-docs.netlify.app/docs/ci-integrations, which might make plain `frontend.yaml` easier to digest.)

wdyt?